lib/filterx/object-json-array.c

```diff
@@ -172,6 +172,25 @@ _set_subscript(FilterXList *s, guint64 index, FilterXObject *new_value)
   return TRUE;
 }
 
+static gboolean
+_unset_index(FilterXList *s, guint64 index)
+{
+  FilterXJsonArray *self = (FilterXJsonArray *) s;
+
+  if (G_UNLIKELY(index >= JSON_ARRAY_MAX_SIZE))
+    return FALSE;
+
+  self->super.super.modified_in_place = TRUE;
```

Comment:
A `json_object_array_del_idx()` seems to be missing from here.

Reply:
This is a rebase issue, thanks for noticing.

Reply:
fixed. I've also reviewed my latest iteration against the @alltilla's original branch I started from and I nothing stood out.